notebooks/Notebook1.md

```diff
@@ -2,33 +2,80 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
+Fill in this part with a summary and reflection on the critique you received for
 last week's work. Answer questions such as:  How, specifically, did the feedback help
 improve the project? Did the feedback point out or offer something you hadn't considered?
 Did it help you make a design decision? Was it helpful in addressing the most pressing
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+I thought that the critique was very good for giving me clear points on what I
+should focus on and further consider when implementing my language. One example
+that the feedback pointed out which I had not considered was creating a clear
+ranking of what functionality I want to prioritize. While I know I wanted to
+prioritize some things over others, I had not created a list. I think this was
+particularly useful feedback at this stage of the project because having a
+clearer plan will allow me to ensure I am staying on track later and not falling
+into time-sink issues as we discussed in class so that what I am working on is
+crucial functionality and not unnecessary to the goals for my DSL.
+
```

Comment:
I think this is a very good plan. Also moving forward like we discussed in class it might be nice to come up some example of your natural language and map out how those would play into particular numpy functions.

Reply:
Did you end up creating a prioritized list of features? If so, that could be a good thing to include / link to in the notebook!